### PR TITLE
add inline-image class and set a basic style

### PR DESCRIPTION
**Pull request by scottaubrey**

### CODE REVIEW

**Comment by scottaubrey** [comment]
Consider adding `vertical-align: middle` to `.inline-image` class to match bioRxiv:
Screenshot 2023-05-23 at 10 12 38 am

Current:
Screenshot 2023-05-23 at 10 12 49 am

With vertical-align:
Screenshot 2023-05-23 at 10 12 54 am

File: src/utils/content-to-jsx.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const contentToJsx = (content: Content, index?: number): JSXContent => {
       if (!content.contentUrl) {
         return '';
       }
-      return <img loading="lazy" key={index} src={generateImageUrl(content.contentUrl)}></img>;
+      return <img loading="lazy" {...(content.meta.inline ? { className: 'inline-image' } : {})} key={index} src={generateImageUrl(content.contentUrl)}></img>;
```
Comment (will-byrne) [inline src/utils/content-to-jsx.tsx:48]: can we add a test here to ensure the class is being applied?

**Review by will-byrne — APPROVED** [review]

**Comment by discodavey** [comment]
@scottaubrey Are you happy with the CSS change?

**Comment by scottaubrey** [comment]
> @scottaubrey Are you happy with the CSS change?

I'm not sure you need my approval here - I was just asking if we'd like to do that to match bioRxiv. If you're happy, I am :)

**Pull request merged**